pkg/metrics/engine.go

```diff
@@ -72,6 +73,13 @@ func newEngineMetrics() engineMetrics {
 			Help:      "Engine 'head' operations handling time",
 		})
 
+		streamDuration = prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
hm, i look at it and think if it can be useful as a metric that just answers how long we were acquiring an object stream, but not a streaming duration itself. mb that is ok